frontend/src/components/Item.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { Item } from '../constants/Interfaces';
+import { View, Text, StyleSheet } from 'react-native';
+import { Svg, Defs, ClipPath, Circle, Image } from 'react-native-svg';
+
+
+interface Props {
+    item: Item
+}
+
+let picHeight = 60;
+let picWidth = 60;
+
+// different ways to present the item
+export function ItemTag(props: Props) {
+    let tags = [];
+    let firstTag = true;
+
+    for (let t in props.item.tags) {
+        let tempComp = <Text key={t}>, {props.item.tags[t]}</Text>;
+        if (firstTag) {
+            tempComp = <Text key={t}>{props.item.tags[t]}</Text>;
```

Comment:
What is the difference between the 2 possible `tempComp`s?

Reply:
That was just so the commas for tags will appear correctly. Basically it'll be "tag1, tag2, tag3" instead of ", tag1, tag2, tag3" or "tag1tag2tag3"

Reply:
Also realizing that's a bit dumb so I'll go ahead and change that